src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -66,21 +66,33 @@
  global-interceptors
  (fn [{:keys [db]} [_ {:keys             [url-prefix
                                           enable-stmt-html
-                                          enable-admin-status]
+                                          enable-admin-status
+                                          no-val?]
                        ?oidc             :oidc
                        ?oidc-local-admin :oidc-enable-local-admin}]]
    (merge {:db (assoc db
                       ::db/xapi-prefix url-prefix
                       ::db/enable-statement-html enable-stmt-html
                       ::db/oidc-enable-local-admin (or ?oidc-local-admin false)
                       ::db/enable-admin-status enable-admin-status)}
-          (when ?oidc
-            {:dispatch [:oidc/init ?oidc]}))))
+          (when (or ?oidc no-val?)
+            {:fx [(when ?oidc [:dispatch [:oidc/init ?oidc]])
+                  (when no-val? [:dispatch [:session/proxy-token-init]])]}))))
```

Comment:
Wouldn't `cond->` be the more obvious syntax here?

Reply:
see slack